### DISCUSSION TITLE
Multi-Line Suggestions

### DISCUSSION THREAD

**Discussion by Anselmoo** [community/community#45019] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

In the case of typing multi-lines at once in VSCode, it would be nice for Copilot to also to able to suggest multiple suggestions at once.

<img width="263" alt="image" src="https://user-images.githubusercontent.com/13209783/213886934-d8ba06bd-9428-4160-b6b8-f25227388b3b.png">

See also: https://github.com/community/community/discussions/44563

**Comment by Anselmoo** [comment]
<img width="268" alt="image" src="https://user-images.githubusercontent.com/13209783/213887070-88041b4d-fec1-4bbc-9065-3fefb3959487.png">

**Comment by iamcjwilliams** [comment, marked as answer]
@Anselmoo I will notify our Copilot team! Thank you for your feedback🚀
